src/stories/20_Features/Animations.html

```diff
@@ -0,0 +1,217 @@
+                  animate-in-delay="0.25s"
+                  animate-in-after="image3"
+                  src="https://unsplash.it/720/320/?image=13"
+                  width="720"
+                  height="320" 
+                  layout="responsive"></amp-img>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Combining Animations -->
+    <!-- You can combine multiple animations by nesting them into multiple elements. -->
+    <amp-story-page id="combining-animations">
+      <amp-story-grid-layer template="thirds">
+        <h1>fly-in-left + fade-in</h1>
+        <div animate-in="fly-in-left"                   
+             animate-in-duration="2s"
+             grid-area="middle-third">
+          <div class="square" 
+               animate-in-duration="2s"
+               animate-in="fade-in">
+          </div>
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+      
+    <!-- ## Twirl-In -->
+    <amp-story-page id="twirl-in">
+      <amp-story-grid-layer template="thirds">
+        <h1>twirl-in</h1>
+        <div class="square"
+            animate-in="twirl-in">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Fly-In-Left -->
+    <amp-story-page id="fly-in-left">
+      <amp-story-grid-layer template="thirds">
+        <h1>fly-in-left</h1>
+        <div class="square"
+            animate-in="fly-in-left">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Fly-In-Right -->
+    <amp-story-page id="fly-in-right">
+      <amp-story-grid-layer template="thirds">
+        <h1>fly-in-right</h1>
+        <div class="square"
+            animate-in="fly-in-right">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Fly-In-Top -->
+    <amp-story-page id="fly-in-top">
+      <amp-story-grid-layer template="thirds">
+        <h1>fly-in-top</h1>
+        <div class="square"
+            animate-in="fly-in-top">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Fly-In-Bottom -->
+    <amp-story-page id="fly-in-bottom">
+      <amp-story-grid-layer template="thirds">
+        <h1>fly-in-bottom</h1>
+        <div class="square"
+            animate-in="fly-in-bottom">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Rotate-In-Left -->
+    <amp-story-page id="rotate-in-left">
+      <amp-story-grid-layer template="thirds">
+        <h1>rotate-in-left</h1>
+        <div class="square"
+            animate-in="rotate-in-left">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Rotate-In-Right -->
+    <amp-story-page id="rotate-in-right">
+      <amp-story-grid-layer template="thirds">
+        <h1>rotate-in-right</h1>
+        <div class="square"
+            animate-in="rotate-in-right">
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Drop-In -->
```

Comment:
The menu on the left is not scrollable, so you need to scroll at the end of the page to go through all of them

Reply:
done